packages/service-library/tests/rabbitmq/test_rabbitmq_utils.py

```diff
@@ -10,28 +10,33 @@
         ({"hello": "1", "b": "2"}, "b_2-hello_1"),
     ],
 )
+@pytest.mark.no_cleanup_check_rabbitmq_server_has_no_errors()
```

Comment:
why?

Reply:
My comment above should explain why. I'm not sure why it's like this